### PR TITLE
add --no-pull flag

### PR DESCRIPTION
**Pull request by zacharym**
@thieman @paetling

### CODE REVIEW

File: dusty/cli/up.py [hunk 1]
```diff
@@ -7,12 +7,15 @@
...
 
 Usage:
-  up [--no-recreate]
+  up [--no-recreate] [--no-pull]
 
 Options:
   --no-recreate   If a container already exists, do not recreate
                   it from scratch. This is faster, but containers
                   may get out of sync over time.
+
```
Comment (thieman) [inline dusty/cli/up.py]: No newline here, observe the existing styles

File: dusty/cli/up.py [hunk 2]
```diff
@@ -7,12 +7,15 @@
...
+  --no-pull       Do not pull dusty managed repos from remotes.
+                  Helpful if you do not have internet access.
```
Comment (thieman) [inline dusty/cli/up.py]: This second line is unnecessary I think

File: dusty/cli/up.py [hunk 3]
```diff
@@ -22,4 +25,5 @@
 
 def main(argv):
     args = docopt(__doc__, argv)
-    return Payload(start_local_env, recreate_containers=not args['--no-recreate'])
+    return Payload(start_local_env, recreate_containers=not args['--no-recreate'],
+                   repull_repos=not args['--no-pull'])
```
Comment (thieman) [inline dusty/cli/up.py]: just `pull`

**Comment by paetling** [comment]
seems good.

**Comment by thieman** [comment]
Nits otherwise :bear:

**Pull request merged**